tests/test_dask_image/test_ndmeasure/test_find_objects.py

```diff
@@ -0,0 +1,36 @@
+import dask.array as da
+import numpy as np
+import pandas as pd
+import pytest
+
+import dask_image.ndmeasure
+
+
+@pytest.fixture
+def label_image():
+    """Return small label image for tests.
+
+    dask.array<array, shape=(5, 10), dtype=int64, chunksize=(5, 5), chunktype=numpy.ndarray>
+
+    array([[  0,   0,   0,   0,   0,   0,   0, 333, 333, 333],
+            [111, 111,   0,   0,   0,   0,   0, 333, 333, 333],
+            [111, 111,   0,   0,   0,   0,   0,   0,   0,   0],
+            [  0,   0,   0, 222, 222, 222, 222, 222, 222,   0],
+            [  0,   0,   0,   0,   0,   0,   0,   0,   0,   0]])
+
+    """
+    label_image = np.zeros((5, 10)).astype(int)
+    label_image[1:3,0:2] = 111
+    label_image[3,3:-2] = 222
+    label_image[0:2,-3:] = 333
+    label_image = da.from_array(label_image, chunks=(5, 5))
+    return label_image
+
+
+def test_bounding_boxes(label_image):
+    result = dask_image.ndmeasure.find_objects(label_image).compute().compute()
+    expected = pd.DataFrame.from_dict(
+        {0: {111: slice(1, 3, 1), 222: slice(3, 4, 1), 333: slice(0, 2, 1)},
+        1: {111: slice(0, 2, 1), 222: slice(3, 8, 1), 333: slice(7, 10, 1)}}
```

Comment:
When trying out different input arrays

```python
label_image = np.random.randint(0,100, (3, 3))
label_image = ndimage.zoom(label_image, zoom=3, order=1)
label_image = ndimage.label(label_image>50)[0].astype(int)
label_image = da.from_array(label_image, chunks=(5, 5))
```

I often get (not always, I think when there's at least one merge):
> ValueError: Metadata mismatch found in `from_delayed`.

Reply:
> Just saw that ndimage.find_objects returns slices such as `slice(1,3,None)` (instead of `slice(1,3,1)`)

Ah, that looks like my mistake. I had assumed that the default slice step was 1, but instead the default value should be `None`. So I think we're seeing None anytime the label doesn't cross the border of an array chunk, because I didn't need to alter/combine slice information.

I don't think this particular thing will have made a material difference, but it's easy for me to fix what I'm doing to match a default step size of None.

Reply:
The metadata mismatch problem I still need to fix (thanks for pointing it out!)

Reply:
I'm not able to replicate your metadata mismatch error, @m-albert. Do you have a minimal, reproducible example?

Reply:
I'm getting the error with this code using the latest dask and PR branch.
```python
import numpy as np
import dask.array as da
import dask_image.ndmeasure
from scipy import ndimage

np.random.seed(0)
label_image = np.random.randint(0,100, (3, 3))
label_image = ndimage.zoom(label_image, zoom=3, order=1)
label_image = ndimage.label(label_image>50)[0].astype(int)
label_image = da.from_array(label_image, chunks=(5, 5))

result = dask_image.ndmeasure.find_objects(label_image).compute()
```
With `np.random.seed(1)` instead it runs through

Reply:
Hm, I am able to replicate this bug today. I'll have another look at it now.

Reply:
Ok, it seems the problem happens when at least one of the label image chunks has no non-zero labels in it. 

That means there is an empty dask array within the `arrays` list, which then doesn't play well with the `_find_objects` function (I think it's the `dd.merge(df1, df2, ...` line that's the problem, still investigating).

Reply:
Update: the problem is actually with `_find_objects` in this line `df1 = dd.from_delayed(df1, meta=meta)`. If df1 is a delayed object of an empty dask dataframe, we get the metadata mismatch error.

Reply:
Found and fixed!

Turns out this was easy to fix - I just had to make sure that the pandas dataframe we create in `_find_bounding_boxes` from a dictionary always has the column names specified. That way if the dictionary is empty (no non-zero labels found in that chunk), we still get the right shape of dataframe.